Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -648,14 +648,14 @@ uint WrapCoord(int coord, uint wrap, int size) {{
               "  float3 coords = float3(float(uv.x) / size_s, float(uv.y) / size_t, layer);\n");
     if (api_type == APIType::OpenGL || api_type == APIType::Vulkan)
     {
-      if (!host_config.backend_sampler_lod_bias)
+      /*if (!host_config.backend_sampler_lod_bias)
```

Comment:
The shader fallback path has been disabled for testing... Will drop this when ready